Add `textAlign` prop to TH and TD (#5420)
jpmorganchase/salt-ds: site/docs/components/table/examples.mdx

> import {
>   LivePreview
> } from "../../../src/components/index";
> ---

Author: ```suggestion
---
```

The `divider` prop can be used to adjust the appearance of the dividers in the table. It accepts: `primary`, `secondary`, `tertiary` or `none`.

<LivePreview componentName="table" exampleName="TFootVariant" />

## Text alignment

You can control the text alignment of table cells and headers using the `textAlign` prop. It accepts: `left` or `right`.

## Best practices

> - Use `textAlign="right"` for numeric data to improve readability.

Author: ```suggestion
-  Numeric data should be right-aligned to improve readability.
```
